src/components/InventoryGroups/Modals/ModalSchemas/schemes.js

```diff
@@ -4,6 +4,7 @@ import componentTypes from '@data-driven-forms/react-form-renderer/component-typ
 import { nameValidator } from '../../helpers/validate';
 import { Text } from '@patternfly/react-core';
 import { getGroups } from '../../utils/api';
+import awesomeDebouncePromise from 'awesome-debounce-promise';
```

Comment:
We could use the fec debounce function. Even though it is just a wrapper around the same thing, it would avoid having to add the package to inventory and possibly deal with version inconsistencies. 

https://github.com/RedHatInsights/frontend-components/blob/master/packages/utils/src/debounce/debounce.ts#L10

Reply:
yeah, I find it useful. I will try to replace it in a separate pr/commit

Reply:
We already using this package in one other place. I think this is could be done in the future improvements :)